pkg/manager/member/utils.go

```diff
@@ -203,3 +203,15 @@ func pdPodName(tcName string, ordinal int32) string {
 func tidbPodName(tcName string, ordinal int32) string {
 	return fmt.Sprintf("%s-%d", controller.TiDBMemberName(tcName), ordinal)
 }
+
+// CombineAnnotations merges two annotations maps
+func CombineAnnotations(a, b map[string]string) map[string]string {
+	m := make(map[string]string)
+	for k, v := range a {
+		m[k] = v
+	}
+	for k, v := range b {
+		m[k] = v
+	}
+	return m
+}
```

Comment:
Maybe use the following code more refine？
```
if a == nil {
    a = make(map[string]string)
}
for k, v := range b {
    a[k] = v
}
return a
```

Reply:
I agree. updated.